admin/js/boldgrid-backup-admin-backup-now.js

```diff
@@ -155,13 +155,10 @@ BOLDGRID.BACKUP.BackupNow = function( $ ) {
 		 * not refresh until we know a backup is in progress so that we know the in progress bar will
 		 * show when the page refreshes.
 		 */
-		if ( 'total-upkeep_page_boldgrid-backup' === pagenow ) {
-			setTimeout( function() {
-				location.reload();
-			}, 6000 );
-		} else {
-			$( 'body' ).trigger( 'boldgrid_backup_initiated' );
```

Comment:
You removed this line. How does that affect the page that is listening for the `boldgrid_backup_initiated` trigger?

Reply:
according to the notes, this was only used when trying to display the in progress container within the modal which was the behavior when not on the Backup Archive's page. However, we have changed this so that the behavior is the same on all pages. It reloads regardless of which page your on, and then displays the progress bar.

Reply:
I put the code back with a slight modification, which will cause the desired effect on the Dashboard > Updates page, as well as all of our pages. Check it out :D